consensus/types/src/consts.rs

```diff
@@ -22,3 +22,19 @@ pub mod altair {
 pub mod merge {
     pub const INTERVALS_PER_SLOT: u64 = 3;
 }
+pub mod deneb {
+    use crate::{Epoch, Uint256};
+
+    use lazy_static::lazy_static;
+
+    lazy_static! {
+        pub static ref BLS_MODULUS: Uint256 = Uint256::from_dec_str(
+            "52435875175126190479447740508185965837690552500527637822603658699938581184513"
+        )
+        .expect("should initialize BLS_MODULUS");
+        pub static ref MIN_EPOCHS_FOR_BLOB_SIDECARS_REQUESTS: Epoch = Epoch::from(4096_u64);
+    }
+    pub const VERSIONED_HASH_VERSION_KZG: u8 = 1;
+    pub const BLOB_SIDECAR_SUBNET_COUNT: u64 = 6;
+    pub const MAX_BLOBS_PER_BLOCK: u64 = BLOB_SIDECAR_SUBNET_COUNT;
```

Comment:
I think we need these to be constant in LH but just wanted to note that these are preset and config values in the spec and not constants. Probably ok now as mainnet and minimal use the same values, maybe worth adding a comment?

`MAX_BLOBS_PER_BLOCK` in Deneb preset:
https://github.com/ethereum/consensus-specs/blob/c5c7233e49e86d4ee386cc4735ebe3f81807ea73/presets/mainnet/deneb.yaml#L10

`BLOB_SIDECAR_SUBNET_COUNT` in Deneb config:
https://github.com/ethereum/consensus-specs/blob/c5c7233e49e86d4ee386cc4735ebe3f81807ea73/configs/mainnet.yaml#L142

Reply:
Really good point. 

> BLOB_SIDECAR_SUBNET_COUNT

Looks like we didn't extend [these](https://github.com/sigp/lighthouse/pull/4426) changes to deneb 

> MAX_BLOBS_PER_BLOCK

The issues surrounding this one are likely similar to the ones described in [this PR](https://github.com/sigp/lighthouse/issues/4562) 

I'll work on trying to sort these out

Reply:
updated these here: https://github.com/sigp/lighthouse/pull/4841